she_codes_news/users/views.py

```diff
@@ -0,0 +1,34 @@
+# Create your views here.
+from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.views.generic.edit import CreateView
+from django.views import generic
+
+from .models import CustomUser
+from .forms import CustomUserCreationForm
+from news.models import NewsStory
+
+
+class CreateAccountView(CreateView):
+    form_class = CustomUserCreationForm
+    success_url = reverse_lazy("login")
+    template_name = 'users/createAccount.html'
+
+class AccountView(generic.DetailView):
+    model = CustomUser
+    template_name = "users/account.html"
+    context_object_name = "user"
+    # context_object_name = "news_newsstory"
+
+    def get_object(self):
+        return self.request.user
+
```

Comment:
you could have used this view to produce an author profile view if you didn't hard code it to only work with the authenticated user